client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/components/credentials-form.tsx

```diff
@@ -29,21 +29,31 @@ export const CredentialsForm: FC< CredentialsFormProps > = ( { onSubmit, onSkip
 		isPending,
 		submitHandler,
 		importSiteQueryParam,
+		getContinueButtonText,
 	} = useCredentialsForm( onSubmit );
 
 	const queryError = useQuery().get( 'error' ) || null;
 
+	let errorMessage =
+		queryError === 'ticket-creation'
+			? translate( 'We ran into a problem submitting your details. Please try again shortly.' )
+			: null;
+
+	errorMessage =
+		isEnglishLocale && errors.root && errors.root.type !== 'manual' && errors.root.message
+			? errors.root.message
+			: errorMessage;
+
```

Comment:
I feel this could be a bit more readable by using if/else statements. WDYT?
```suggestion
	let errorMessage;
	if ( isEnglishLocale && errors.root && errors.root.type !== 'manual' && errors.root.message ) {
		errorMessage = errors.root.message;
	} else if ( queryError === 'ticket-creation' ) {
		errorMessage = translate(
			'We ran into a problem submitting your details. Please try again shortly.'
		);
	}
```